src/main/web/src/scenes/Profile/ProfilePage.css

```diff
@@ -0,0 +1,79 @@
+.profile-page {
```

Comment:
Div `profile-page` is not necessary, `page` already determines everything from this, the height is not required